telemetry/definitions/commonDefinitions.json

```diff
@@ -855,6 +855,71 @@
             "name": "userChoice",
             "type": "string",
             "description": "User selection from a predefined menu (not user-provided input)" 
+        },
+        {
+            "name": "awsqAssignClickCount",
+            "type": "int",
+            "description": "Number of times users have triggered /assign" 
```

Comment:
is there a time frame or session under which this `awsqAssignClickCount` will be updated?